docs/UserGuide.md

```diff
@@ -50,110 +70,181 @@ Some example commands you can try:
+| STATUS    | s/     | The status to label for a show                        |
+| TAG       | t/     | The tag to label a show                               |
+| COMMENT   | c/     | The comment to describe a show                        |
+| RATE      | r/     | The rating to give a show                             |
 
 <div markdown="block" class="alert alert-info">
 
 **:information_source: Notes about the command format:**<br>
 
-* Words in `<UPPER_CASE>` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/<NAME>`, `<NAME>` is a parameter which can be used as `add n/John Doe`.
+* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
+  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/Sex and the City`.
 
 * Items in square brackets are optional.<br>
-  e.g `n/<NAME> s/<STATUS> [t/<TAG>]` can be used as `n/ReZero s/completed t/Anime` or as `n/ReZero s/completed`.
+  e.g `n/NAME s/STATUS [t/TAG]` can be used as `n/ReZero s/completed t/Anime` or as `n/ReZero s/completed`.
 
 * Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[t/<TAG>]…​` can be used as ` ` (i.e. 0 times), `t/Anime`, `t/Sitcom t/Kdrama` etc.
+  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/Anime`, `t/Sitcom t/Kdrama` etc.
+
+* A whitespace must be included before every prefix.<br>
+  e.g. `n/Knives Out t/Suspense` is acceptable but `n/Knives Outt/Suspense` is not.
 
 * Parameters can be in any order.<br>
-  e.g. if the command specifies `n/<NAME> [t/<TAG>]`, `[t/<TAG>] n/<NAME>` is also acceptable.
+  e.g. if the command specifies `n/NAME [t/TAG]`, `[t/TAG] n/NAME` is also acceptable.
 
-* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `n/You n/Me`, only `n/Me` will be taken.
+* All **names must be unique** and duplicates will be ignored.<br>
+  e.g. if you try to add `n/Inception` into the show list that already contains that show, there will be a message telling you that this show already exists in the list.
+
+* If a parameter is expected only once in the command, but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
+  e.g. if you specify `n/Batman n/Superman`, only `n/Superman` will be taken.
 
 * Extraneous parameters for commands that do not take in parameters (such as `exit` and `list` ) will be ignored.<br>
   e.g. if the command specifies `exit 123`, it will be interpreted as `exit`.
 
+* The **index** parameter provided should be a [non-zero unsigned integer](https://en.wikipedia.org/wiki/Integer_(computer_science)) within the allowed range of Java’s [`int`](#glossary) data type. On top of that, the index should be within the bounds of the show list.<br>
+  e.g. If there are 5 shows saved in the show list, then the valid index ranges from 1 to 5. 
+
```

Comment:
Maybe this could be in each of the commands that has an index? This will make it easier for users to know about the warnings in each section of the commands that has an index.

Reply:
Yes that is a good idea.